docs/modules/comments.md

```diff
@@ -0,0 +1,494 @@
+
+#### `foreignCommentsCount`
+
+Returns the number of comments left by other users.
+
+**Arguments**
+
+| Argument Name     | Description                                                     |
+| ----------------- | --------------------------------------------------------------- |
+| `treeId`          | The unique identifier of a comment tree.                        |
+| `parentCommentId` | The identifier of a comment for which to obtain child comments. |
+| `maxLevel`        | The maximum nesting level of the comments to load.              |
+
+#### `allCommentsPaginate`
+
+Returns comments with pagination.
+
+**Arguments**
+
+| Argument Name | Description                       |
+| ------------- | --------------------------------- |
+| `itemsOnPage` | The number of comments on a page. |
+| `pageNumber`  | The number of the page to load.   |
+
+## Client API
+
+The comments ([@resolve-js/module-comments](https://www.npmjs.com/package/@resolve-js/module-comments)) package exports renderless React+Redux components that you can use implement your own UI for user comments. The following components are available:
+
+| Component Name                                                      | Description                                                                      |
+| ------------------------------------------------------------------- | -------------------------------------------------------------------------------- |
+| [`CommentsNotificationRenderless`](#commentsnotificationrenderless) | A component used to display a notification about a new comment.                  |
+| [`RefreshHelperRenderless`](#refreshhelperrenderless)               | A component that allows the application to refresh comments without page reload. |
+| [`CommentsPaginateRenderless`](#commentspaginaterenderless)         | A container used to render comments in a paginated view.                         |
+| [`CommentsTreeRenderless`](#resolvernames)                          | A container used to render comments in a hierarchical view.                      |
+
+### `CommentsNotificationRenderless`
+
+A component that displays a notification about new comments.
+
+**Usage**
+
+```js
+import React from 'react'
+import { CommentsNotificationRenderless } from '@resolve-js/module-comments'
+
+export const CommentsNotification = (props) => (
+  <CommentsNotificationRenderless
+    treeId="treeId"
+    parentCommentId="parentCommentId"
+    authorId="authorId"
+    checkInterval={10000} // default = 30 * 1000
+    readModelName="customReadModelName" // default = 'readModelName'
+    resolverName="customResolverName" // default = 'foreignCommentsCount'
+    {...props}
+  >
+    {({ count, onClick }) => {
+      // Replace with your code.
+      if (count === 0) return null
+      return (
+        <div onClick={onClick}>
+          Comments have been updated, refresh the page to see the new comments.
```

Comment:
Is this message editable? If so, can you split it into two sentences?

`Comments have been updated. Refresh the page to see the new comments.`

Reply:
I updated the message.

Reply:
Thank you.